website/screens/components/slider/usage/SliderUsagePage.tsx

```diff
@@ -0,0 +1,71 @@
+import Image from "@/common/Image";
+import { DxcList, DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import HeadingLink from "../../../common/HeadingLink";
+import sliderInput from "./images/slider_input.png";
+import sliderVariants from "./images/slider_variants.png";
+import DocFooter from "../../../common/DocFooter";
+import Figure from "../../../common/Figure";
+
+const SliderUsagePage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Usage</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Visualize the output of the slider as feedback to the user of the current state.
+          </DxcText>
+          <DxcText>
+            As more information can give it to the user, the easier the selection will be.
+          </DxcText>
+          <DxcText>
+            If the value has to be specific, give some resource to the user to fill a precise input,
+            i.e. an input next to the slider.
+          </DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Slider with input</HeadingLink>
```

Comment:
We should keep the order of the sections we have in the guidelines: first `Variants`, then `Slider with input`..

Reply:
That was my first thought too, but the text in Slider with input makes more sense in this position. I was thinking of having the section in this position until the text is changed.

Reply:
I said that because it refers to the Slider variants. What do you think @aweell ?

Reply:
Can you elaborate on "makes more sense" @Jialecl?

I agree with @marcialfps that we should keep the order of the sections given, then we can even discuss if we should call variants the `discrete` and `continuous` forms of the slider.

Reply:
Based on the text I made the correlation to the usage in which you needed a specific value and also it seems to be introducing the variants.

Reply:
I've just read the text and makes sense what you propose for how the section paragraph is redacted (Or copied & pasted from the previous documentation 👎🏼 ).

But still, just to be consistent between components, keep the sections order as:
* Usage 
* Variants
* XYZ...

And I note in the doc issue that the paragraph needs to be re-redacted